camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/ConfigInheritanceYamlTest.java

```diff
@@ -326,53 +320,72 @@ public void testOverridesParentConfig() throws Exception {
                 "- type: org.apache.brooklyn.entity.stock.BasicApplication",
                 "  brooklyn.config:",
                 "    shell.env:",
-                "      ENV1: myEnv1",
+                "      ENV: myEnv",
+                "      ENV3: myEnv",
                 "    templates.preinstall:",
                 "      "+emptyFile.toUri()+": myfile",
+                "      "+emptyFile3.toUri()+": myfile",
                 "    files.preinstall:",
                 "      "+emptyFile.toUri()+": myfile",
+                "      "+emptyFile3.toUri()+": myfile",
                 "    templates.install:",
                 "      "+emptyFile.toUri()+": myfile",
+                "      "+emptyFile3.toUri()+": myfile",
                 "    files.install:",
                 "      "+emptyFile.toUri()+": myfile",
+                "      "+emptyFile3.toUri()+": myfile",
                 "    templates.runtime:",
                 "      "+emptyFile.toUri()+": myfile",
+                "      "+emptyFile3.toUri()+": myfile",
                 "    files.runtime:",
                 "      "+emptyFile.toUri()+": myfile",
+                "      "+emptyFile3.toUri()+": myfile",
                 "    provisioning.properties:",
                 "      mykey: myval",
+                "      mykey3: myval",
                 "      templateOptions:",
                 "        myOptionsKey: myOptionsVal", 
+                "        myOptionsKey3: myOptionsVal", 
                 "  brooklyn.children:",
                 "  - type: org.apache.brooklyn.entity.software.base.EmptySoftwareProcess",
                 "    brooklyn.config:",
                 "      shell.env:",
                 "        ENV2: myEnv2",
+                "        ENV3: myEnv2",
                 "      templates.preinstall:",
                 "        "+emptyFile2.toUri()+": myfile2",
+                "        "+emptyFile3.toUri()+": myfile2",
```

Comment:
Minor...

Why are you reusing the name "myfile2" for the target filename? Is that because you want to check it doesn't complain if two files are being uploaded to the same place? Or just because you don't care about the value?

If you don't care about the value, I'd prefer it to say "myfile3" as that makes the test slightly clearer (and ensures we haven't mixed up the key-values).

Same goes for "myEnv2" etc.